config/application.yml

```diff
@@ -0,0 +1,11 @@
+# Add configuration values here, as shown below.
```

Comment:
- clone 1 file from this file, named `config/application.yml.example`
- file `config/application.yml` move into `.gitignore`
- remove this file `config/application.yml` using command `git rm --cached config/application.yml`